ethereumetl/cli/stream.py

```diff
@@ -68,7 +69,7 @@ def stream(last_synced_block_file, lag, provider_uri, output, start_block, entit
 
     streamer_adapter = EthStreamerAdapter(
         batch_web3_provider=ThreadLocalProxy(lambda: get_provider_from_uri(provider_uri, batch=True)),
-        item_exporter=create_item_exporters(output),
+        item_exporter=create_item_exporter(output),
```

Comment:
Please use `create_item_exporters` instead of `create_item_exporter`

Reply:
Reverted that change, thanks @medvedev1088